api/docs/triager.dox

```diff
@@ -36,9 +36,11 @@
 
 We have a rotating Triager who is responsible for monitoring our continuous testing infrastructure and for triaging incoming requests from users.  Specific duties include:
 
-- Ensure that our Continuous Integration testing through Github Actions and Jenkins are operating smoothly.
+- Ensure that our Continuous Integration testing through Github Actions are operating smoothly.
+  - Do this by going to the Github Actions page: https://github.com/DynamoRIO/dynamorio/actions.
   - If flaky tests are failing too often, assign someone to fix them ASAP, or mark them to be ignored in `runsuite_wrapper.pl`.
 - Watch merges to master for failures on the longer test suite.
+  - Do this by watching for merges on the dynamorio-devs@ list.
```

Comment:
No need to look at the devs list.  These are on the same Actions page linked above, to find details -- but the summary page is the green check or red X by each commit at https://github.com/DynamoRIO/dynamorio/commits/master/ .  That's what the goal is: to have that page contain nothing but green checks.